internal/logstore/observability/lokistack/lokistack_test.go

```diff
@@ -0,0 +1,231 @@
+package lokistack
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	obs "github.com/openshift/cluster-logging-operator/api/observability/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var _ = Describe("MigratePipelinesForLokiStack", func() {
+	var (
+		spec obs.ClusterLogForwarderSpec
+	)
+	BeforeEach(func() {
+		spec = obs.ClusterLogForwarderSpec{
+			Outputs: []obs.OutputSpec{
+				{
+					Name: "lokistack-out",
+					Type: obs.OutputTypeLokiStack,
+					LokiStack: &obs.LokiStack{
+						Target: obs.LokiStackTarget{
+							Name:      "internal-lokistack",
+							Namespace: "openshift-logging",
+						},
+						Authentication: &obs.HTTPAuthentication{
+							Token: &obs.BearerToken{
+								ServiceAccount: &corev1.LocalObjectReference{
+									Name: "lokistack-writer-sa",
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+	})
+
+	It("single tenant - single lokiStack output", func() {
```

Comment:
Can we write this as... `It("should .... ` with the expectation of the outcome

Reply:
I moved these tests to the loki migration tests and used table tests instead.